src/components/WeatherWidgetCurrentData.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <section class="weather-info">
+    <div class="weather-info__image-wrapper">
+      <img
+        class="weather-info__image"
+        :src="props.weatherIcon"
+        :alt="`${props.weatherDescription} Weather Image`"
+      />
+      <p class="weather-info__description">
+        {{ props.weatherDescription }}
+      </p>
+    </div>
+    <div class="weather-info__temperature-wrapper">
+      <h3 class="weather-info__temperature-info">
+        {{ props.temperature }}&deg;C
+      </h3>
+    </div>
+    <ul class="weather-info__additional-info">
+      <li class="weather-info__additional-info-item">
+        Wind: {{ props.windSpeed }}
+      </li>
+      <li class="weather-info__additional-info-item">
+        Pressure: {{ props.pressure }}
+      </li>
+      <li class="weather-info__additional-info-item">
+        UV Index: {{ props.uvIndex }}
+      </li>
```

Comment:
we can loop over those properties to save some space in the template and make it more scalable

Reply:
Agreed. We should loop over an array. The titles "Wind, Pressure, UV Index" should also come from that array.